src/ewig/draw.cpp

```diff
@@ -23,6 +23,7 @@
 #include <scelta.hpp>
 
 extern "C" {
+#define _XOPEN_SOURCE_EXTENDED
 #include <ncurses.h>
```

Comment:
Hmmm, I think I had this at some point but then removed it. Can't remember why. Probably differences between the Mac ncurses and the Nix provided one. Maybe it doesn't hurt.

Reply:
I see it failing in the nix build: /nix/store/q2q1sg5sljia8sihhwcpbxir70yw33bw-glibc-2.27-dev/include/features.h:204:0: note: this is the location of the previous definition
 # define _XOPEN_SOURCE_EXTENDED 1

Probably needs an #ifndef

Reply:
:+1: I'll investigate

Reply:
( I've since pushed a new revision that adds the #ifndef )